source/shared/net/references/developer-tools.rst

```diff
@@ -0,0 +1,35 @@
+.. _developer-tools:
+
+===============
+Developer Tools
+===============
+
+.. contents::
+   :local:
+   :backlinks: none
+
+
+Decrypt encrypted output
+========================
+
+Some Concordium tools, such as the Mobile Wallet, sometimes produce encrypted output. These files can be decrypted and inspected using the **utils** tool . You can download the tool from **reference here**.
+
+To encrypt, enter:
+
+``utils encrypt --in input.blah --out output.json``
```

Comment:
Maybe you could choose a better name than `input.blah` that I chose, perhaps `input.example`.